test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/MathTest.scala

```diff
@@ -304,6 +310,49 @@ class MathTest {
     test(Double.MinPositiveValue, -3.423e-319)
   }
 
+  @Test def ulpForFloat(): Unit = {
```

Comment:
I'm not sure what to say that would be interesting to write. It's just that this method does not rely on arithmetic operations, and so is always accurate. There are many methods that are always accurate on Floats (for example `nextUp` and friends), and for which we have no such comment. If I add a comment here, it might be confusing why the other methods don't have that comment.